### ISSUE TITLE
Can't install with latest elasticsearch version

### ISSUE THREAD

**Issue by aeryaguzov** [imotov/elasticsearch-analysis-morphology#27]
```
Plugin [elasticsearch-analysis-morphology] is incompatible with Elasticsearch [2.1.1]. Was designed for version [2.1.0]
```

And there is no version for 2.1.1, is there a way to install plugin for 2.1.1 ?

**Comment by MaxJust** [comment]
+1 the same problem

**Comment by imotov** [comment]
@aeryaguzov @MaxJust The plugin needs to be updated and rereleased to work with 2.1.1. I am going to be working on this during Christmas vacation. So, it will be most likely available next week.

**Comment by imotov** [comment]
Released plugin for v2.1.1 and v2.0.2.

**Comment by MaxJust** [comment]
thx, very mach.
